src/SFA.DAS.EmployerIncentives.Database/Tables/IncentiveApplication.sql

```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [dbo].[IncentiveApplication]
```

Comment:
Do these tables need indexing or is this just a write store?

Reply:
No read yet.  There will be at some point but we can add the index then.